apps/prairielearn/assets/scripts/instructorAssessmentManualGradingAssessmentQuestionClient.ts

```diff
@@ -144,20 +144,35 @@ onDocumentReady(() => {
           sortable: true,
           switchable: false,
           formatter: (_value: number, row: InstanceQuestionRowWithIndex) =>
-            html`<a
+            html`
+              <a
                 href="${urlPrefix}/assessment/${row.assessment_question
                   .assessment_id}/manual_grading/instance_question/${row.id}"
+                >Instance ${row.index + 1}</a
               >
-                Instance ${row.index + 1}
-                ${row.open_issue_count
-                  ? html`<span class="badge badge-pill badge-danger">${row.open_issue_count}</span>`
-                  : ''}
-              </a>
+              ${row.open_issue_count
+                ? html`
+                    <a
+                      href="#"
```

Comment:
Would it make sense for this to be a link to the issues page? Possibly using the issue badge component introduced in #10277 (assuming you can merge this one after navbar is reviewed)?

Reply:
IMO no; the issues will be immediately visible on the grading page anyways, and I also think we don't have the ability to hotlink to a view of the issues page that includes issues only for a particular instance question.

Reply:
Makes sense.